packages/@aws-cdk/aws-ecs/lib/cluster.ts

```diff
@@ -55,6 +55,25 @@ export interface ClusterProps {
   readonly capacityProviders?: string[];
 
   /**
+<<<<<<< Updated upstream
+=======
+   * The cluster default capacity provider strategy. This takes the form of a list of CapacityProviderStrategy objects.
+   *
+   * For example
+   * [
+   *  {
+   *    capacityProvider: 'FARGATE',
+   *    base: 10,
+   *    weight: 50
+   *  }
+   * ]
+   *
+   * @default null
+   */
+  readonly defaultCapacityProviderStrategy?: CapacityProviderStrategy[];
```

Comment:
actually - should these be part of this PR? I don't see how this is related to the deployment alarms changes.

Reply:
Sir that's true, but when I did push the PR directly after working on my perious task which is related to defaultCapacityproviders, there is so much clumsy in PR 

So I did following steps, To keep the extraneous commits to a minimum in this and other PRs. It looks like there’s only one commit that has to do with the data structures and constructor method, so the PR branch should only contain that commit and any future ones I make.

1.pull down the latest main branch from aws-cdk origin
2.rename your current branch to something else (abr-backup or similar)
3.cherry-pick that one commit onto a new branch named “alarambasedrollback” (original name of this current branch)
4.force push the new branch to your fork of the cdk to get rid of the extra stuff


When I did that that commit is also coming to this PR, I don't know why but it appears.